nginx.conf

```diff
@@ -0,0 +1,49 @@
+server {
+    listen 80;
+    server_name localhost;
+    location / {
+        proxy_pass https://gafm-srv:443;
+        proxy_set_header Host $host;
+        proxy_set_header X-Real-IP $remote_addr;
+        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+        proxy_set_header X-Forwarded-Proto $scheme;
+    }
+}
+
+server {
+    listen 8080;
+    server_name localhost;
+    location / {
+        proxy_pass https://gafm-srv:443;
+        proxy_set_header Host $host;
+        proxy_set_header X-Real-IP $remote_addr;
+        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+        proxy_set_header X-Forwarded-Proto $scheme;
+    }
+}
+
+server {
+    listen 443;
+    server_name localhost;
+    ssl_protocols TLSv1 TLSv1.1 TLSv1.2;
+    location / {
+        proxy_pass https://gafm-srv:443;
+        proxy_set_header Host $host;
+        proxy_set_header X-Real-IP $remote_addr;
+        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+        proxy_set_header X-Forwarded-Proto $scheme;
+    }
+}
+
+server {
+    listen 8443;
+    server_name localhost;
+    ssl_protocols TLSv1 TLSv1.1 TLSv1.2;
+    location / {
+        proxy_pass https://gafm-srv:443;
+        proxy_set_header Host $host;
+        proxy_set_header X-Real-IP $remote_addr;
+        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+        proxy_set_header X-Forwarded-Proto $scheme;
+    }
+}
```

Comment:
Looks like there's a whitespace error. From the top-level directory of the project, please run the following:

```
$ pip install pre-commit
$ pre-commit install
```